eduNEXT/tutor-contrib-picasso
# fix: improve the commands output

## Description

This PR adds `click.echo` in the `enable-private-packages` and `enable-themes` commands to make debugging easier when they are inside the `extra-commands` command. Also, check for themes to be dictionaries to avoid the `type ignore` comments.

## How to test

- If the tests pass, the commands work as expected (See the integration test for more info).
- Run a build process with this version of contrib-picasso and search for the logs related to the `enable-private-packages` and `enable-themes` commands.
